src/main/java/net/kaczmarzyk/spring/data/jpa/utils/Converter.java

```diff
@@ -136,6 +136,8 @@ public <T> T convert(String value, Class<T> expectedClass, boolean ignoreCase) {
 			return (T) convertToEnum(value, (Class<? extends Enum<?>>) expectedClass, ignoreCase);
 		} else if (expectedClass.isAssignableFrom(Date.class)) {
 			return (T) convertToDate(value);
+                } else if (expectedClass.isAssignableFrom(Calendar.class)) {
```

Comment:
Please remove support of `Calendar` conversion from the scope of this PR or :
 1) add tests for `string` => `Calendar` conversion  
 2) extend README.md with information about new supported conversion: `README.md, section: Supported conversions`
3) update CHANGELOG.md

Reply:
What can I write in CHANGELOG.md?

Reply:
```
* Added conversion support for `Calendar`
* Fixed the bug with redundant joins
```